WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.swift

```diff
@@ -137,15 +137,9 @@ class NotificationsViewController: UIViewController, UIViewControllerRestoration
     /// Used by JPScrollViewDelegate to send scroll position
     internal let scrollViewTranslationPublisher = PassthroughSubject<Bool, Never>()
 
-    /// The last time when user seen notifications
-    var lastSeenTime: String? {
-        get {
-            return userDefaults.string(forKey: Settings.lastSeenTime)
-        }
-        set {
-            userDefaults.set(newValue, forKey: Settings.lastSeenTime)
-        }
-    }
+    lazy var viewModel: NotificationsViewModel = {
+        NotificationsViewModel(userDefaults: userDefaults)
+    }()
```

Comment:
Nit: Maybe  `viewModel` should be marked as private to ensure it can't be modified externally.